lua/autorun/nl_library.lua

```diff
@@ -0,0 +1,609 @@
+-- The tazer is no longer a dependancy :^) --
+AllowedWake = { -- these conditions can be removed when a friend presses +use
+	["ZZZ"] = true,
+	["---"] = true,
+	["PLV"] = false
+}
+
+NLL = {}
+
+if SERVER then
+
+util.AddNetworkString("nllinfopayload") -- establish payload netstring
+
+hook.Add("PlayerSay", "TG_NLL", function(ply, str)
+	if ply.nllismuted then return "" end
+end)
+
+hook.Add("PlayerCanSeePlayersChat", "TG_NLL", function(text, teamOnly, listener, talker)
+	if talker.nllismuted then
+		return false
+	end
+end)
+
+hook.Add("PlayerCanHearPlayersVoice", "TG_NLL", function(listener, talker)
+	if talker.nllismuted then
+		return false,false
+	end
+end)
+
+NLL.ConditionStorage = {} -- Magical table that stores all players with conditions. Better than ply.condition
+
+function NLL.GetCondtion(ply) -- Function that returns the player's table value or "NIL" as string.
+	return NLL.ConditionStorage[ply] or "NIL"
+end
```

Comment:
Why is the second return a string? Better yet, why is there a second return at all?